lib/tutorials/en_US/serving-files.md

```diff
@@ -1,91 +1,81 @@
 ## Serving static files
 
-_This tutorial is compatible with hapi v16_
+_This tutorial is compatible with hapi v17_
 
 Inevitably while building any web application, the need arises to serve a simple file from disk. There's a hapi plugin called [inert](https://github.com/hapijs/inert) that adds this functionality to hapi through the use of additional handlers.
 
 First you need to install and add inert as a dependency to your project:
 
 `npm install --save inert`
 
-## reply.file()
+## `h.file(path, [options])`
 
-First, to use the reply method:
+Firstly, let's see how to use the [`h.file()`](https://github.com/hapijs/inert#hfilepath-options) method:
 
 ```javascript
-server.register(require('inert'), (err) => {
+const start = async () => {
 
-    if (err) {
-        throw err;
-    }
+    await server.register(require('inert'));
 
     server.route({
         method: 'GET',
         path: '/picture.jpg',
-        handler: function (request, reply) {
-            reply.file('/path/to/picture.jpg');
+        handler: function (request, h) {
+
+            return h.file('/path/to/picture.jpg');
         }
     });
 
-    server.start((err) => {
+    await server.start();
 
-        if (err) {
-            throw err;
-        }
+    console.log('Server running at:', server.info.uri);
+};
 
-        console.log('Server running at:', server.info.uri);
-    });
-});
+start();
 ```
 
-As I'm sure you've guessed, in its simplest form you pass a path to `reply.file()` and you're done.
+As you can see above, in its most basic form you pass a path to `reply.file()` and you're done.
 
 ### Relative paths
 
-To simplify things, especially if you have multiple routes that respond with files, you can configure a base path in your server and only pass relative paths to `reply.file()`
+To simplify things, especially if you have multiple routes that respond with files, you can configure a base path in your server and only pass relative paths to `reply.file()`:
```

Comment:
I think this is the only other reference to `reply.file()` that oughtta be `h.file()` 👍